README.md

```diff
@@ -1,20 +1,220 @@
+        'networkConnectionMbps' => 1000,
+        'internetConnectionMbps' => 1000,
+        'description' => 'Test Description',
+    ])
+    ->toJson();
+
+header("Content-Type: application/json"); 
+
+echo $jsonOutput;
 ```
 
-**Write a "calling script"** called "index.php" that that uses the library to build the list of JSON events. Look at the several [php examples](examples) to help get you started. The example below shows a simple `index.php` file using PDO to connect to a MySQL database.
+Single Venue with multiple Events example:
+
+```php
+<?php
+
+use DateTime;
+use LanPartyPublisherPhp\Event;
+use LanPartyPublisherPhp\Publisher;
+
+$publisher = Publisher::make()
+    ->createOrganisation('Test Organisation')
+    ->createVenue('Test Venue', [
+        'gpsLatitude' => null,
+        'gpsLongditude' => null,
+    ]);
+
+$events = [
+    [
+        'name' => 'Test Event 1',
+        'opts' => [
+            'start' => DateTime::createFromFormat('Y-m-d H:i:s', '2021-01-01 00:00:00'),
+            'finish' => DateTime::createFromFormat('Y-m-d H:i:s', '2021-01-02 00:00:00'),
+            'seatsTotal' => 100,
+            'seatsAvailable' => 50,
+            'ticketsOnSale' => 'Yes',
+            'ticketCurrencyIso4217' => 'GBP',
+            'ticketPriceInAdvance' => 10.99,
+            'ticketPriceOnDoor' => 15.99,
+            'isTicketsOnSale' => true,
+            'sleeping' => 1,
+            'hasShowers' => true,
+            'isAlcoholAllowed' => true,
+            'hasSmokingArea' => true,
+            'networkConnectionMbps' => 1000,
+            'internetConnectionMbps' => 1000,
+            'description' => 'Test Description',
+        ],
+    ],
+    [
+        'name' => 'Test Event 2',
+        'opts' => [
+            'start' => DateTime::createFromFormat('Y-m-d H:i:s', '2021-01-01 00:00:00'),
+            'finish' => DateTime::createFromFormat('Y-m-d H:i:s', '2021-01-02 00:00:00'),
+            'seatsTotal' => 100,
+            'seatsAvailable' => 50,
+            'ticketsOnSale' => 'Yes',
+            'ticketCurrencyIso4217' => 'GBP',
+            'ticketPriceInAdvance' => 10.99,
+            'ticketPriceOnDoor' => 15.99,
+            'isTicketsOnSale' => true,
+            'sleeping' => 1,
+            'hasShowers' => true,
+            'isAlcoholAllowed' => true,
+            'hasSmokingArea' => true,
+            'networkConnectionMbps' => 1000,
+            'internetConnectionMbps' => 1000,
+            'description' => 'Test Description',
+        ],
+    ]
+];
+
+$events = array_map(function (array $value) {
+    return Event::make($value['name'], $value['opts']);
+}, $events);
+
+$publisher->addEvents('Test Venue', $events);
+
+header("Content-Type: application/json"); 
 
-**index.php** - Start with the example code; [withPdoMySQLDatabase.php](examples/withPdoMySQLDatabase.php)
+echo $publisher->toJson();
 
-Adapt the above code to your own database structure, and view the resulting JSON in your browser.
+```
```

Comment:
Could you describe a bit better, why to remove the examples directly and move all the code into the README?